### PR TITLE
add time series outputs

### PR DESCRIPTION
**Pull request by mariano-recurve**
### Your checklist for this pull request

Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

- [X] Make sure you are requesting to **pull a feature/bugfix branch** (right side). Don't request your master!
- [X] Make sure tests pass and coverage has not fallen `./test.sh`.
- [X] Update the [CHANGELOG.md](../CHANGELOG.md) to describe your changes in a bulleted list under the "Development" section at the top of the changelog. If this section does not exist, create it.
- [X] Make sure code style follows PEP 008 using `black`.
- [X] Make sure that new functions and classes have inline docstrings.

### Description

Add ability to fetch time series cost outputs (hourly for electricity, quarterly for gas)

### CODE REVIEW

File: flexvalue/cli.py [hunk 1]
```diff
@@ -249,3 +249,115 @@ def valid_utility_climate_zone_combos(utility, version):
...
+
+@cli.command()
+@click.option("-v", "--version", default="2020", show_default=True)
+@click.option(
+    "--url-prefix",
+    default="https://storage.googleapis.com/flexvalue-public-resources/db/v1/",
+    show_default=True,
+)
+@click.option("--skip-if-exists/--overwrite-if-exists", default=False)
+def download_avoided_costs_data_db(url_prefix, version, skip_if_exists):
```
Comment (jtucci1210) [inline flexvalue/cli.py]: Why are we duplicating this method in this file?

File: flexvalue/cli.py [hunk 2]
```diff
@@ -249,3 +249,115 @@ def valid_utility_climate_zone_combos(utility, version):
...
+    show_default=True,
+    help="What version of the avoided costs data to use in the analysis",
+)
+@click.option(
+    "--output-filepath",
+    default=".",
+    show_default=True,
+    help="Filepath to where the example inputs will be stored",
+)
+def generate_example_inputs(output_filepath, version):
```
Comment (jtucci1210) [inline flexvalue/cli.py]: This also looks like an accidental duplication?

File: flexvalue/db.py [hunk 3]
```diff
@@ -161,11 +161,7 @@ def get_filtered_acc_gas(database_version, start_year, end_year):
     -------
     pd.DataFrame
     """
-    columns = [
-        "year",
-        "month",
-        *ACC_COMPONENTS_GAS,
-    ]
+    columns = ["year", "month", *ACC_COMPONENTS_GAS]
```
Comment (jtucci1210) [inline flexvalue/db.py:164]: interesting that the formatter changed this back

File: flexvalue/calculations.py [hunk 4]
```diff
@@ -682,9 +690,30 @@ def get_results(self, user_inputs):
...
         elec_benefits = self.get_all_trc_electricity_benefits_df(user_inputs)
         gas_benefits = self.get_total_trc_gas_benefits(user_inputs)
-
         # if index wasn't already set with the ID colum, set it for joining to the output
         if user_inputs.index.name != "ID":
             user_inputs = user_inputs.set_index("ID")
         outputs_table = user_inputs.join(outputs_table).reset_index()
         return outputs_table, outputs_table_totals, elec_benefits, gas_benefits
+
+    def get_time_series_results(self, user_inputs):
```
Comment (jtucci1210) [inline flexvalue/calculations.py:699]: so then the thought is here that a user would call either the get_results method OR the get_time_series_results method right?

**Review by jtucci1210 — CHANGES_REQUESTED** [review]
Overall looks good but I think there may be an issue with duplicating some of the methods on the cli.py

File: flexvalue/db.py (hunk 3 above)
Comment (mariano-recurve) [reply flexvalue/db.py:164]: Yeah, not sure what to make of that!

File: flexvalue/calculations.py (hunk 4 above)
Comment (mariano-recurve) [reply flexvalue/calculations.py:699]: That is correct yes.  I considered adding them to get_results, and in fact get_results already returns the time series elec_benefits, but for some reason it sums up the gas_benefits to a total.  I would have had to add another output to get_results to include the time series gas outputs and I was worried about breaking existing code in the wild.

**Review by jtucci1210 — APPROVED** [review]
lgtm!

**Pull request merged**